src/dotnet/OpenAPI.Validator/Properties/Resources.resx

```diff
@@ -360,4 +360,7 @@
   <data name="XmsParameterLocation" xml:space="preserve">
     <value>The parameter '{0}' is defined in global parameters section without 'x-ms-parameter-location' extension. This would add the parameter as the client property. Please ensure that this is exactly you want. If so, apply the extension "x-ms-parameter-location": "client". Else, apply the extension "x-ms-parameter-location": "method".</value>
   </data>
+  <data name="LongRunningOperationsOptionsValidatorMessage" xml:space="preserve">
+    <value>A LRO Post operation with return value must have "x-ms-long-running-operation-options" extension enabled.</value>
```

Comment:
should it be "with return schema"? or "type" or "model"?

Reply:
Please update documentation at https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/openapi-authoring-automated-guidelines.md

Reply:
could we indicate which LRO operation in the message and maybe which return type? "LRO operation X wich returns model Y must have..."?

Reply:
Changed to return schema. Will add the documentation. Such a message is not necessary because operation will be post which you are already indication and return schema is not important.